release.md

```diff
@@ -28,12 +28,12 @@ Branches on the conda-forge featuretools repo are automatically built and the pa
     ```bash
     git remote add upstream https://github.com/conda-forge/featuretools-feedstock.git
     ```
-4. If you made the fork previously and its master branch is missing commits, update it with any changes from upstream
```

Comment:
Unsure if we will be able to rename this branch since it is part of conda-forge, worth looking into

Reply:
It seems that conda-forge defaults to master, and most of the packages use master.
- https://conda-forge.org/docs/maintainer/adding_pkgs.html
- https://github.com/conda-forge